### DISCUSSION TITLE
Full cable trace

### DISCUSSION THREAD

**Discussion by dfilippi** [netbox-community/netbox#9689] [General]
Hi all,
I have several workstation connected to their ip phone, in this way

PC -> (lan) IP PHONE (wan) -> (rear) patch panel (front) -> switch interface.

I've replicated this to netbox and when I go to the switch port and click trace on the interface I see only

switch interface -> patch panel -> ip phone.

I lost information about pc.
Is there any way to show in the trace all the interfaces?

**Comment by candlerb** [comment]
I'm afraid not.  "lan" and "wan" on the IP phone are interfaces, not front/rear ports.

It's like tracing through a switch - in fact, is *is* a two-port switch.  But Netbox cable tracing is for physical cables only, not going through vlans in a switch.  Front port and rear port are physically directly connected, so is just another cable segment.

**Reply by jeremystretch** [reply]
On NetBox v3.1 or later, you can create a bidirectional bridge relationship between the two phone interfaces, which the cable trace will follow in either direction. (Note that this works only with a one-to-one bridge.)
